UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/RootBodyPartContainer.cs

```diff
@@ -302,6 +302,11 @@ public virtual void ImplantPeriodicUpdate()
 			{
 				prop.ImplantPeriodicUpdate();
 			}
+			if(IsBleeding)
+			{
+				float TotalBloodLossFromLostLimbs = LimbLossBleedingValue;
```

Comment:
what's  the need for this assignation if you're using it right in the next line without any alterations from ``LimbLossBleedingValue``?

Reply:
good point. Done.